lib/end-points.js

```diff
@@ -1,130 +1,222 @@
 /* jslint maxlen: 200 */

```

Comment:
This file is auto generated via https://github.com/confuser/node-dotmailer/blob/master/lib/end-point-builder.js

Reply:
oh right!

Reply:
will review, verify the endpoints and update

Reply:
re-generated - nice touch btw!

Reply:
Looks like the builder needs updating to put the `,` in the correct place, sorry about that 😞

Reply:
yeah no worries - i get the comment about manual steps now ;-)